hack/test-cmd.sh

```diff
@@ -98,6 +91,54 @@ export OPENSHIFT_PROFILE="${CLI_PROFILE-}"
 # Begin tests
 #
 
+# test client not configured
+[ "$(osc get services 2>&1 | grep 'no server found')" ]
+
+# Set KUBERNETES_MASTER for osc from now on
+export KUBERNETES_MASTER="${API_SCHEME}://${API_HOST}:${API_PORT}"
+
+# Set certificates for osc from now on
+if [[ "${API_SCHEME}" == "https" ]]; then
+    # test bad certificate
+    [ "$(osc get services 2>&1 | grep 'certificate signed by unknown authority')" ]
+
+    # ignore anything in the running user's $HOME dir
+    export HOME="${CERT_DIR}/admin"
+fi
+
+# test config files from the --config flag
+osc get services --config="${CERT_DIR}/admin/.kubeconfig"
+
+# test config files from env vars
+OPENSHIFTCONFIG="${CERT_DIR}/admin/.kubeconfig" osc get services
+KUBECONFIG="${CERT_DIR}/admin/.kubeconfig" osc get services
+
+# test config files in the current directory
+TEMP_PWD=`pwd` 
```

Comment:
Doesn't seem to pass with $OS_ROOT, I'll leave it like that for now and fix later.